cetmix_tower_server/views/cx_tower_command_view.xml

```diff
@@ -164,6 +165,7 @@
                     options="{'color_field': 'color'}"
                     optional="hide"
                 />
+                <field name="secret_ids" widget="many2many_tags" optional="hide" />
```

Comment:
Please make this field visible only to the members of the "Manager" and "Root" groups.

Reply:
Visible != accessible